packages/shared-component--hero/src/hero.pcss

```diff
@@ -5,268 +5,200 @@
   padding-bottom: 6rem;
-
-  @media (--mq-large) {
-    padding-bottom: 13rem; /* 72px vertical space */
+}
+@media (min-width: 64em) {
+  .coop-c-hero--pull .coop-c-hero__inner {
+    padding-bottom: 13rem;
   }
 }
-
 .coop-c-hero__title {
   position: relative;
   z-index: 2;
   margin: 0;
-  font-size: 2.875rem;
+  font-size: var(--type-h1-l);
   line-height: 3.625rem;
-
-  @media (--mq-medium) {
+}
+@media (min-width: 48em) {
+  .coop-c-hero__title {
     float: left;
     width: 66.6666%;
-    padding-right: 1rem;
+    padding-right: var(--spacing-16);
   }
 }
-
-.coop-c-hero--media .coop-c-hero__title {
-  @media (--mq-medium) {
+@media (min-width: 48em) {
+  .coop-c-hero--media .coop-c-hero__title {
     width: 50%;
   }
 }
-
 .coop-c-hero__media {
-  margin: 1rem -1rem 0;
+  margin: var(--spacing-16) -1rem 0;
   position: relative;
   z-index: 2;
-
-  @media (--mq-medium) {
+}
+@media (min-width: 48em) {
+  .coop-c-hero__media {
     width: 50%;
     margin: 0;
-    padding-left: 1rem;
+    padding-left: var(--spacing-16);
     float: right;
   }
 }
-
 .coop-c-hero__image {
   display: block;
-
-  & img {
-    display: block;
-    width: 100%;
-  }
 }
-
+.coop-c-hero__image img {
+  display: block;
+  width: 100%;
+}
 .coop-c-hero__body {
   position: relative;
   z-index: 2;
   padding-top: 1.25rem;
-
-  @media (--mq-medium) {
+}
+@media (min-width: 48em) {
+  .coop-c-hero__body {
     float: left;
     width: 66.6666%;
-    padding-right: 2rem;
-  }
-
-  @media (--mq-xlarge) {
-    padding-top: 2rem;
+    padding-right: var(--spacing-32);
   }
-
-  & p {
-    font-size: 1.375rem;
-    line-height: 2rem;
-    margin-bottom: 2rem;
-
-    /* required to overwrite foundation CSS */
-    &:last-child {
-      margin-bottom: 0;
-    }
+}
+@media (min-width: 81.25em) {
+  .coop-c-hero__body {
+    padding-top: var(--spacing-32);
   }
 }
-
-.coop-c-hero--media .coop-c-hero__body {
-  @media (--mq-medium) {
+.coop-c-hero__body p {
+  font-size: var(--type-lp-s);
+  line-height: var(--spacing-32);
+  margin-bottom: var(--spacing-16);
+}
+.coop-c-hero__body p:last-child {
+  margin-bottom: 0;
+}
+@media (min-width: 48em) {
+  .coop-c-hero--media .coop-c-hero__body {
     width: 50%;
   }
 }
-
 .coop-c-hero__cta__link {
   display: inline-block;
-  padding: var(--spacing-base--1-4) calc(var(--spacing-base--1-2) * 1.25);
+  padding: 0.5rem 1.25rem;
   color: var(--color-white);
-  background: var(--color-button-primary);
-  font-size: 1.125rem;
-  font-family: var(--font-family);
+  background: var(--color-button-green-primary);
+  font-size: var(--type-sp-l);
+  font-family: Avenir-Next, Helvetica Neue, Helvetica, Arial, sans-serif;
   font-weight: 500;
   text-align: center;
   border: 0;
   border-radius: 0.4375rem;
   transition: all 0.15s linear;
   outline: none;
   text-decoration: none;
-
-  @media (--mq-large) {
-    font-size: 1.25rem;
+}
+@media (min-width: 64em) {
+  .coop-c-hero__cta__link {
+    ont-size: var(--type-sp-l);
```

Comment:
Missing `f` on font

Reply:
But probably can be swapped with `coop-btn` in the HTML

Reply:
css removed